src/lib.rs

```diff
@@ -721,9 +721,8 @@ impl Subscription {
 impl Drop for Subscription {
     fn drop(&mut self) {
         if self.do_unsub {
-            match self.unsub() {
-                Ok(_) => {}
-                Err(_) => {}
+            if let Err(error) = self.unsub() {
+                eprintln!("error unsubscribing during Subscription Drop: {:?}", error);
```

Comment:
Not sure this makes sense, but understand since we have other println. Production lib should not do println IMO.

Reply:
I agree that it's preferable to avoid printing. What are the highest priority feedback mechanisms to support? Table stakes for rust libraries is to use the log crate, which has 0 performance impact unless the user attaches a log receiver of some sort (env_logger is what many people use as a default). Do you see things like prometheus metrics being a priority? I'm used to avoiding logs almost entirely for this stuff and using metrics instead since it keeps monitoring cheap and easy to aggregate and analyze, but the log crate is probably a good minimum thing to support that can replace this asap.

Reply:
Client libs should not print IMO.. ;)

Reply:
But we can leave in for now, but next up want to do `last_error()` or something like it and remove all prints.

Reply:
In production people often feed it into log aggregators rather than just dumping to stderr/stdout. The nice thing about the log crate is it doesn't specify anything about how the log lines are consumed, only produced. So whatever a client wants to do with possible error messages, they will plug into the source and do whatever they want with the info. It's like NATS but for Rust library feedback. A nice way to decouple the production of messages from their consumption.

Reply:
Agree, but client libs should only surface errors, apps can log all they want.. ;)